Sources/FronteggSwift/embedded/FronteggWebView.swift

```diff
@@ -14,10 +14,11 @@ import AuthenticationServices
 public struct FronteggWebView: UIViewRepresentable {
     public typealias UIViewType = WKWebView
     private var fronteggAuth: FronteggAuth
+    private let loginHint: String?
```

Comment:
move loginHint to fronteggAuth Service as we have for pendingLink option

Reply:
Done